save-frontend/src/main/kotlin/com/saveourtool/save/frontend/App.kt

```diff
@@ -291,6 +301,15 @@ class App : ComponentWithScope<PropsWithChildren, AppState>() {
                                         withRouterLink = true
                                     }
                                 }
+
+                                Route {
+                                    path = "/${FrontendRoutes.NOT_FOUND.path}"
```

Comment:
Why do we need a fixed Not Found page?
We wanted to leave an invalid url in the browser path if the page is not found.

Reply:
Deleted this

Reply:
Then `FrontendRoutes.NOT_FOUND` is also not needed?

Reply:
Redone